index.html

```diff
@@ -80,7 +80,7 @@ <h2 class="second-header">I'm a Software Developer</h2>
   <!-- My works area -->
   <section class="work-section">
     <div class="section-title-container">
-      <h2 class="second-title">My recent Works</h2>
```

Comment:
- Text Headings: The page uses heading tags (h1, h2) to structure the content. However, the heading hierarchy could be improved to be more meaningful and consistent. For example, the section titles are using h2 tags while the work titles are also using h2 tags, which could cause confusion for screen reader users. 
-- Changes suggested.
-- We fixed it.